backend/apps/sitemap/views/snapshots.py

```
@@ -0,0 +1,23 @@
+from rest_framework.views import APIView
+from rest_framework.response import Response
+from owasp.models.snapshot import Snapshot
+
+class SnapshotsSitemapView(APIView):
+    """Sitemap view for snapshots."""
+
+    def get(self, request, *args, **kwargs):
```

Ruff (0.12.2)

Missing docstring in public method

(D102)